src/commands/extension/install.ts

```diff
@@ -19,5 +22,6 @@ export default class Install extends BaseCommand {
 
   async run() {
     await installExtension(this, this.flags.dumpCode, this.flags.noCache);
+    await fetchRemoteTypes();
```

Comment:
not security: what are the performance/UX implications of doing a remote fetch on every install? `extension:install` is potentially used repeatedly in a short interval as a developer iterates on an extension. would there be an easy way to do this only after a certain amount of time has passed, since clearly we don't expect this to change minute by minute? 

I guess this does happen _after_ the install, which is nice, but the fact it shows a message every time is potentially distracting/confusing.

Reply:
If a developer wants to make many changes in a short period, they should use `watch` instead of repeatedly calling `install`. I think we should keep it simple (no cache) unless we hear complaints.